src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor.cs

```diff
@@ -178,6 +183,8 @@ protected override void OnAfterRender(bool firstRender)
     {
         if (firstRender)
         {
+            LogViewerViewModel.LogEntries.MaximumEntryCount = Options.CurrentValue.Frontend.ConsoleLogHistoryLimit;
```

Comment:
Since this VM is now injected from DI, its ctor could have `IOptionsMonitor<DashboardOptions>`. The max could be set in the VM ctor without requiring console logs to wire them up. Seems a bit more encapsulated to me.

Reply:
Agree, that is nicer.